website/src/content/docs/docs/libraries/typespec-client-generator-core/guideline.md

```diff
@@ -181,6 +181,8 @@ A [`SdkBuiltInType`](../reference/js-api/interfaces/sdkbuiltintype/) represents
 
 [`SdkModelType`](../reference/js-api/interfaces/sdkmodeltype/) type represents a TCGC model type. It is typically converted from a TypeSpec [`Model`](https://typespec.io/docs/language-basics/models/) type.
 
+[`SdkModelPropertyType`](../references/js-api/interfaces/sdkmodelpropertytype/) type represents a TCGC model property type. It is typically converted from a TypeSpec [`ModelProperty`](https://typespec.io/docs/standard-library/reference/js-api/interfaces/modelproperty/) type. It represents a property of a model. It has `flatten`, `additionalProperties` to indicate if the property could be flatten and the model could accept additional properties with specific type. If a model is a discriminated model, `discriminatorProperty` is the property that is used as a discriminator and `discriminatedSubtype` is the list of all the subtypes of this discriminated model. For all sub types of the discriminated model, `discriminatorValue` is the instance value for the discriminator for this sub type.
```

Comment:
Could you help to rephrase this one which is a bit difficult for me to understand.   Does SdkModelPropertyType have the value of `flatten`, `additionalProperties` or  `discriminatorProperty`?   
How about using a separate line to describe each case?

Reply:
They are the properties of `SdkModelPropertyType`. I refined the whole doc with AI and fixed lots of expression and link issues.